mac/macdeploy.sh

```diff
@@ -7,6 +7,7 @@
 
 MACDEPLOYQT="macdeployqt"
 LIBRARY_NAME="libQupZilla.2.dylib"
+PLUGINS="QupZilla.app/Contents/Resources/plugins"
```

Comment:
Hmm, can you please upload somewhere the `QupZilla.app` after running macdeployqt on it?